integration-tests/vault/src/test/java/io/quarkus/vault/VaultSysITCase.java

```diff
@@ -27,7 +27,7 @@
 
 import io.quarkus.test.QuarkusUnitTest;
 import io.quarkus.test.common.QuarkusTestResource;
-import io.quarkus.vault.runtime.client.VaultClientException;
+import io.quarkus.vault.client.VaultClientException;
```

Comment:
that would be an api change?

Reply:
Yeah. I couldn't get around this without duplicating the name. Obviously, we need `VaultClientException` to be in the client module. The only solution I could think of was to derive a copy of in the runtime but then I realized it would never be thrown by the client itself.

When I was thinking would be 2 PR's (one for adding the standalone client and one for replacing the in-extension client) we could do a 3.5 and then 4.0 release to match "breaking" changes. Currently it looks like we need to release a 4.0 client and have end-users deal with the minor changes.